packages/runtime/container-runtime/src/summarizerHeuristics.ts

```diff
@@ -20,6 +20,12 @@ export class SummarizeHeuristicData implements ISummarizeHeuristicData {
         return this._lastSuccessfulSummary;
     }
 
+    public numSystemOps: number = 0;
+    private numSystemOpsBefore: number = 0;
```

Comment:
What do the before fields mean? Could you please add a comment.

Reply:
These variables store the op counts at the beginning of a summarization attempt. They are used to adjust the op counts appropriately once a summarization is completed.